CHANGELOG.md

```diff
@@ -12,9 +12,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 
 ### Removed
+- Remove explicit play() on video element for Safari browser
```

Comment:
Instead of this can we describe the actually issue that you fix so when we look at it later we know why we remove this.

Reply:
Updated